recipe/meta.yaml

```diff
@@ -31,7 +31,7 @@ about:
   license: GPL-2.0-only
   summary: Converts a Python dictionary or other native data type into a valid XML string.
   license_family: GPL
-  license_file: LICENSE.txt
+  license_file: LICENCE.txt
```

Comment:
The file name has to be consistent with https://github.com/quandyfactory/dicttoxml/blob/master/LICENCE.txt

Reply:
I don't get it, but i can't fix until tomorrow, so please go ahead with any changes

Reply:
Oh spelling